syntax = "proto3";

package stats;

option go_package = "model/stats";

message Data {
	string Candidate = 1;
	bool Gender = 2;
	int32 Age = 3;
	string Region = 4;
	string Job = 5;
	string Education = 6;
	bool Married = 7;
	bool Divorced = 8;
	bool HasCar = 9;
	string HouseType = 10;
	int32 Salary = 11;
	bool HasDebt = 12;
	string Ideology = 13;
	bytes VoterID = 14;
	int64 Number = 15;
}

message List {
	repeated string Data = 1;
}

message NumberList {
	repeated int64 Data = 1;
}